qemu/tests/balloon_check.py

```diff
@@ -62,11 +62,20 @@ def memory_check(self, step, ballooned_mem):
         error.context("Check memory status %s" % step, logging.info)
         mmem = self.get_ballooned_memory()
         gmem = self.get_memory_status()
-        guest_ballooned_mem = abs(gmem - self.ori_gmem)
-        if (abs(mmem - self.ori_mem) != ballooned_mem or
-                (abs(guest_ballooned_mem - ballooned_mem) > 100)):
-            self.error_report(step, self.ori_mem - ballooned_mem, mmem, gmem)
-            raise error.TestFail("Balloon test failed %s" % step)
+        if ballooned_mem >= self.ori_mem - 100:
```

Comment:
Used for balloon_illegal test, when i set  windows's balloon value in (1,100), guest's free memory should < 50M( a minumum free mem suggested by virtio-win team )
i set the "if" for the above condition.